scrollingpagerindicator/src/main/java/ru/tinkoff/scrollingpagerindicator/ScrollingPagerIndicator.java

```diff
@@ -131,6 +131,43 @@ public void setDotColor(@ColorInt int color) {
         invalidate();
     }
 
+    /**
+     * @return the dot size
+     */
+
+    public int getDotSize() {
+        return dotNormalSize;
+    }
+
+    /**
+     * Sets dot size
+     *
+     * @param size dot color
+     */
+    public void setDotSize(int size) {
```

Comment:
What is the reason for this feature? Do you need dynamically change dot sizes?

Reply:
yes, because I cannot use xml and I have to change the size of dots dynamically in code. The default size is not ok for my slider.